pkg/networkserver/internal/test/shared/application_uplink_queue.go

```diff
@@ -276,6 +321,7 @@ func handleApplicationUplinkQueueTest(ctx context.Context, q ApplicationUplinkQu
 		), should.BeTrue),
 
 		!a.So(assertAdd(ctx,
+			genericApp1Ups[1],
 			genericApp1Ups[2],
```

Comment:
Previous test specification implictly assumes that our queue is strictly FIFO, and this may not be the case for multiple consumers.

Updated test to reflect this, simply by avoiding mixing uplinks from multiple applications.